realsense_camera/launch/r200_nodelet_modify_params.launch

```diff
@@ -8,20 +8,21 @@
 
   <!-- These 'arg' tags are just place-holders for passing values from test files.
   The recommended way is to pass the values directly into the 'param' tags. -->
-  <arg name="enable_depth"      default="true" />
-  <arg name="enable_ir"         default="false" />
-  <arg name="enable_ir2"        default="false" />
-  <arg name="enable_color"      default="true" />
-  <arg name="enable_pointcloud" default="false" />
-  <arg name="enable_tf"         default="true" />
-  <arg name="enable_tf_dynamic" default="false" />
-  <arg name="mode"              default="manual" />
-  <arg name="depth_width"       default="640" />
-  <arg name="depth_height"      default="480" />
-  <arg name="color_width"       default="1920" />
-  <arg name="color_height"      default="1080" />
-  <arg name="depth_fps"         default="30" />
-  <arg name="color_fps"         default="30" />
+  <arg name="enable_depth"        default="true" />
+  <arg name="enable_ir"           default="false" />
+  <arg name="enable_ir2"          default="false" />
+  <arg name="enable_color"        default="true" />
+  <arg name="enable_pointcloud"   default="false" />
+  <arg name="enable_tf"           default="true" />
+  <arg name="enable_tf_dynamic"   default="false" />
+  <arg name="tf_publication_rate" default="1.0" />
```

Comment:
This file is missing setting the parameter with the argument value.
Around line 34, the following should be added:
```
  <param name="$(arg camera)/driver/tf_publication_rate" type="double" value="$(arg tf_publication_rate)" />
```